cache-all.xq

```diff
@@ -5,6 +5,7 @@ xquery version "1.0";
 import module namespace aws_config = "http://history.state.gov/ns/xquery/aws_config" at 'modules/aws_config.xqm';
 import module namespace bucket = 'http://www.xquery.co.uk/modules/connectors/aws/s3/bucket' at 'modules/xaws/modules/uk/co/xquery/www/modules/connectors/aws-exist/s3/bucket.xq';
 
+declare namespace httpclient="http://exist-db.org/xquery/httpclient";

```

Comment:
The `httpclient` namespace isn't used in this module.

Reply:
my mistake, this is an old commit that should not have made it into this pr. I will throw away the PR and send a new one